### DISCUSSION TITLE
Olá

### DISCUSSION THREAD

**Discussion by Vandiiinhoo1** [ThallesLazaro/GunboundTH-Server-and-Game-Portable#2] [General]
Consegue fazer funcionar os exitem ?

**Comment by ThallesLazaro** [comment]
> Consegue fazer funcionar os exitem ?

Desculpe pela demora hahahaha, não estou acostumado com esse sistema de perguntas do github, só vi sua pergunta agora...
Então, ate o momento estou no foco de fazer o servidor ser compatível com servidores públicos, ou seja, poder colocar o servidor com um ip publico e jogar com qualquer pessoa que tiver o IP e o client.
